### DISCUSSION TITLE
最近移动香港直连的是不是出问题了，移动测下来好像没有低于100的了

### DISCUSSION THREAD

**Discussion by jfplyl999** [XIU2/CloudflareSpeedTest#334] [讨论]

**Comment by XIU2** [comment]
最近这段时间 Cloudflare 的 香港节点频繁维护，你可以在官方网站看到状态：
Hong Kong - (HKG) Re-routed
https://www.cloudflarestatus.com/

**Comment by jfplyl999** [comment]
谢谢

**Comment by cxw620** [comment]
> 最近这段时间 Cloudflare 的 香港节点频繁维护，你可以在官方网站看到状态： Hong Kong - (HKG) Re-routed https://www.cloudflarestatus.com/

确实, 以前我广东移动可以跑满带宽, 现在不太可以. 三天两回Re-routed. 现在还是rerouted

**Comment by jfplyl999** [comment]
> 最近这段时间 Cloudflare 的 香港节点频繁维护，你可以在官方网站看到状态： Hong Kong - (HKG) Re-routed https://www.cloudflarestatus.com/

Hong Kong - (HKG) 好像已经正常了，但是移动宽带延时还是180+，不像之前的50+了
